gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-rc-1-all.zip
```

Comment:
Is there a reason why we depend on a release candidate of Gradle here?

Reply:
I was feeling optimistic about being able to target 7.5 after the RC came out.

Now that it's been 3+ weeks with no release or RC2, that may have been overly optimistic…

Reply:
so shall we go with 7.4 and get this merged then?

Reply:
I'm fine with merging this if we go for the latest **released version** right now :upside_down_face: